<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationWidget</class>
 <widget class="QWidget" name="CommunicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>205</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CommunicationWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Containers */

QWidget#CommunicationWidget {
	background-color: transparent;
}

QWidget#chatControlsWidget {
	background-color: rgba(34,34,34,191);
}

QWidget#chatContentWidget {
	background-color: transparent;
}

QWidget#imContentWidget {
	background-color: transparent;
	background-image: url('./data/ui/images/chat/IM_bground1.png');
	background-position: top left;
	background-repeat: no-repeat;
}

/* Buttons */

QPushButton#viewModeButton {
	border: 0px;
	background-color: transparent;
	background-image: url('./data/ui/images/chat/uibutton_HISTORY_normal.png');
	background-position: top left;
	background-repeat: no-repeat;
}

QPushButton#viewModeButton::hover {
	border: 0px;
	background-image: url('./data/ui/images/chat/uibutton_HISTORY_hover.png');
}

QPushButton#viewModeButton::pressed {
	border: 0px;
	background-image: url('./data/ui/images/chat/uibutton_HISTORY_click.png');
}

QPushButton#imButton {
	border: 0px;
	background-color: transparent;
	background-image: url('./data/ui/images/chat/uibutton_IM_normal.png');
	background-position: top left;
	background-repeat: no-repeat;
}

QPushButton#imButton::hover {
	border: 0px;
	background-image: url('./data/ui/images/chat/uibutton_IM_hover.png');
}

QPushButton#imButton::pressed {
	border: 0px;
	background-image: url('./data/ui/images/chat/uibutton_IM_click.png');
}

/* Misc */

QScrollBar {
	background-color: rgba(34,34,34,191);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="mainVerticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="chatContentWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="contentContainerLayout">
            <property name="margin">
             <number>6</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="chatControlsWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="chatHorizontalLayout">
            <item>
             <widget class="QLineEdit" name="chatLineEdit">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid rgba(255,255,255,150); border-radius: 3px; padding-left: 2px;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="viewModeButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>62</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>62</width>
                <height>23</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="imContentWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>103</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>103</width>
           <height>32</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="imButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>84</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>84</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
